docs/content/en/docs/implementing/evaluatemetrics.md

```diff
@@ -1,5 +1,148 @@
 ---
-title: Evaluate metrics
-description: Define all workloads and checks associated with an application
+title: Keptn Metrics
+description: Implement Keptn site metrics
 weight: 130
 ---
+
+The Keptn Metrics Operator provides a single entry point
+to all metrics in the cluster
+and allows you to define metrics based on multiple data platforms
+and multiple instances of any data platform.
+Metrics are fetched independently
+and can be used for an evaluation at workload- and application-level.
+
+This data can be displayed on Grafana
+or another standard dashboard application that you configure
+or can be retrieved using standard Kubernetes commands.
+
+For an introduction to Keptn metrics, see
+[Getting started with Keptn metrics](../getting-started/metrics).
+
+## Keptn metric basics
+
+Keptn metrics are implemented with two resources:
+
+* [KeptnMetric](../yaml-crd-ref/metric.md) --
+  define the metric to report
+* [KeptnMetricsProvider](../yaml-crd-ref/metricsprovider.md) --
+  define the configuration for a data provider
+
+A Metric can be created with a query that fetches data for a property
```

Comment:
I do not understand this sentence, can you explain?

Reply:
This is something Andi mentioned in his video.  I don't understand it either and if we don't give details, this is really pretty useless.  I deleted the sentence.